runtime-api/src/lib.rs

```diff
@@ -266,15 +266,16 @@ sp_api::decl_runtime_apis! {
 		fn retrieve_data(query_id: QueryId, timestamp: Timestamp) -> Option<Value>;
 	}
 
-	pub trait TellorGovernance<AccountId: Codec, Amount: Codec, BlockNumber: Codec, DisputeId: Codec, QueryId: Codec, Timestamp: Codec, Value: Codec, VoteCount: Codec, VoteId: Codec> where
+	pub trait TellorGovernance<AccountId: Codec, Amount: Codec, BlockNumber: Codec, DisputeId: Codec, QueryId: Codec, Timestamp: Codec, Value: Codec> where
 	{
-		/// Determines if an account voted for a specific dispute.
+		/// Determines if an account voted for a specific dispute round.
 		/// # Arguments
 		/// * `dispute_id` - The identifier of the dispute.
+		/// * `vote_round` - The vote round.
 		/// * `voter` - The account of the voter to check.
 		/// # Returns
-		/// Whether or not the account voted for the specific dispute.
-		fn did_vote(dispute_id: DisputeId, voter: AccountId) -> bool;
+		/// Whether or not the account voted for the specific dispute round.
+		fn did_vote(dispute_id: DisputeId, vote_round: u32, voter: AccountId) -> bool;
```

Comment:
Changing `vote_round` to `u8` I think per your last comment

Reply:
Done